tensorflow/core/util/gpu_launch_config.h

```diff
@@ -275,10 +277,12 @@ inline Gpu2DLaunchConfig GetGpu2DLaunchConfig(int xdim, int ydim,
       grid_x, std::min(max_blocks / grid_x, std::max(ydim / block_rows, 1)), 1);
   return config;
 }
+#ifndef TENSORFLOW_USE_ROCM
 inline Cuda2DLaunchConfig GetCuda2DLaunchConfig(int xdim, int ydim,
```

Comment:
Looks like this is already aliased in line 374 in the new file. I might have left this one behind when re-doing my changes after chsigg's. So it looks like we should just remove it.

Reply:
I thought about removing it, but was not sure about it.

There exists both a non-template version (line 253) and a template version (line 366) of the routine `GetGpu2DLaunchConfig`.

The `GetCuda2DLaunchConfig` wrapper that we create on line 374 will cover both the template and non-template version, I think, but was not sure.  So decided to create an explicit wrapper for the non-template version. 

will leave it upto @chsigg to decide whether or not to leave this change in. I can remove the explicit wrapper for the non-template routine if that works better for upstreaming.

Reply:
Either way is fine with me. I'll start the merge as is. Eventually those should all be removed again anyway.